### ISSUE TITLE
the last line of funtion build_model()

### ISSUE THREAD

**Issue by NemoCpp** [Azure/DataScienceVM#9]
the last line of funtion build_model() is 
'''
    model.add(Activation('softmax'))

    # compile and fit model, reduce epochs if you want a result faster
    # the validation set is used to identify parameter settings (epoch) that achieves 
    # the highest classification accuracy

    return model
'''
should it be 
'''
    model.add(Dense(num_classes))
    model.add(Activation('softmax'))

    # compile and fit model, reduce epochs if you want a result faster
    # the validation set is used to identify parameter settings (epoch) that achieves 
    # the highest classification accuracy

    return model
''

**Comment by xiaoyongzhu** [comment]
Thanks! This looks like a bug. I think we were using model.add(Dense(num_classes, activation='softmax')) but looks like we didn't add the Dense layer there.

Thanks for reporting this!
